packages/plugin-core/src/commands/pods/ImportObsidianCommand.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  getAllImportPods,
+  MarkdownImportPod,
+  podClassEntryToPodItemV4,
+} from "@dendronhq/pods-core";
+import vscode from "vscode";
+import { PickerUtilsV2 } from "../../components/lookup/utils";
+import { DENDRON_COMMANDS } from "../../constants";
+import { PodQuickPickItemV4 } from "../../utils/pods";
+import { CommandOpts, ImportPodCommand } from "../ImportPod";
+
+/**
+ * Convenience command that uses the same flow as {@link ImportPodCommand} for
+ * Markdown Pod but simplifies the steps by not requiring the user to fill out a
+ * config.yml file.
+ */
+export class ImportObsidianCommand extends ImportPodCommand {
```

Comment:
can we also have this just be a dedicated command?  import pod -> obsidian

Reply:
Er - you mean 'dedicated pod' instead of being a dedicated command?

If that's what you meant - I think that might require more tweaking of how the import pod flow works, which is what I was trying to avoid in this 80/20 iteration.

Reply:
got it. lets go ahead with this then